### PR TITLE
[qa] wallet*.py: Check for salvagewallet regressions 

### PR DESCRIPTION
**Pull request by maflcko**
This checks for the bug reported in #2480 and #8300.

### CODE REVIEW

**Comment by paveljanik** [comment]
`wallet.py` fails tests.

**Comment by MarcoFalke** [comment]
This requires at least #8324

**Comment by laanwj** [comment]
Re-triggered travis after merging #8324

**Comment by MarcoFalke** [comment]
> WIP tag can go?

You can merge it if you want to, but I'd prefer to extend the test further after #8323.

File: qa/rpc-tests/wallet.py [hunk 1]
```diff
@@ -330,7 +330,12 @@ def run_test (self):
             '-zapwallettxes=2',
             # disabled until issue is fixed: https://github.com/bitcoin/bitcoin/issues/7463
             # '-salvagewallet',
+            # enabled to check for regression: https://github.com/bitcoin/bitcoin/issues/2480
```
Comment (laanwj) [inline qa/rpc-tests/wallet.py]: this conflicts with the above comment

Comment (MarcoFalke) [inline qa/rpc-tests/wallet.py]: This is on purpose. Not sure how to proceed.

**Comment by MarcoFalke** [comment]
Don't consider this pull a blocker for rc1. It should test the regression well enough for now but there might be things to improve and work on later. You can merge it now or leave it for later. I shouldn't matter too much, maybe flip a coin?

**Comment by laanwj** [comment]
Right - going to remove this from the milestone, I don't think we should re-enable a `-salvagewallet` test as long as we have #7463.

**Comment by sipa** [comment]
What is blocking this? It seems #7463 is still open.

**Comment by MarcoFalke** [comment]
Yes, I think this is still relevant. I will put it on my list for 0.15. It seems that I need to catch up with the recent hdwallet changes first. Closing for now.

**Pull request closed without merging**